include/openssl/ssl.h.in

```diff
@@ -236,7 +236,7 @@ typedef struct ssl_cipher_st SSL_CIPHER;
 typedef struct ssl_session_st SSL_SESSION;
 typedef struct tls_sigalgs_st TLS_SIGALGS;
 typedef struct ssl_conf_ctx_st SSL_CONF_CTX;
-typedef struct ssl_comp_st SSL_COMP;
+typedef struct ossl_comp_st SSL_COMP;
```

Comment:
I would not change the typedef from ssl_comp_st to ossl_comp_st. Just move it to openssl/comp.h and keep using SSL_COMP everywhere instead of OSSL_COMP.

Reply:
Those casts from OSSL_COMP to SSL_COMP are awkward and unnecessary if we just do not introduce the new type.